.dockerignore

```diff
@@ -0,0 +1,32 @@
+# Include any files or directories that you don't want to be copied to your
+# container here (e.g., local build artifacts, temporary files, etc.).
+#
+# For more help, visit the .dockerignore file reference guide at
+# https://docs.docker.com/engine/reference/builder/#dockerignore-file
+
+**/.classpath
```

Comment:
This list seems to be more extensive than it strictly necessary for this codebase. Is it an idea to thin it out to avoid that stuff gets ignored, but we're not aware of it?